vnext/Microsoft.ReactNative/Fabric/Composition/ScrollViewComponentView.cpp

```diff
@@ -724,6 +725,10 @@ void ScrollViewComponentView::updateProps(
     updateBackgroundColor(newViewProps.backgroundColor);
   }
 
+  if (!oldProps || oldViewProps.scrollEnabled != newViewProps.scrollEnabled) {
+    m_scrollVisual.ScrollEnabled(newViewProps.scrollEnabled);
```

Comment:
I think you need this here:
    updateHighlight(ScrollbarHitRegion::ArrowFirst);
    updateHighlight(ScrollbarHitRegion::ArrowLast);
    updateHighlight(ScrollbarHitRegion::Thumb);

I suspect the colors will not get updated right away as it is?

Reply:
As you mentioned depending on the setup, either a new scrollview is created or existing scrollview is updated based on the scrollEnabled prop change. Not every scenario updates the scrollbar color accordingly.

Example 1: Existing scrollbar is updated with the change in scrollEnabled
`const Bootstrap: React.FC = () => {
    const [scrollEnabled, setScrollEnabled] = React.useState<boolean>(true);

    return (
        <View>
            <Button title="Toggle" onPress={() => setScrollEnabled(!scrollEnabled)} />
            <ScrollView style={{ height: 75 }} horizontal={false} scrollEnabled={scrollEnabled}>
                <Text>
                   ....
                </Text>
            </ScrollView>
        </View>
    );
};

AppRegistry.registerComponent('Bootstrap', () => Bootstrap);`

Example2: New scrollview is created
`<ScrollView style={{ height: 50 }} horizontal={false} scrollEnabled={false}>
    <Text>
        ...
    </Text>
</ScrollView>`

Modified the updateprops accordingly to support all scenarios. Thanks.